src/backend/joanie/core/serializers/client.py

```diff
@@ -763,7 +763,7 @@ class OrderGroupSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = models.OrderGroup
-        fields = ["id", "is_active", "nb_seats", "nb_available_seats"]
+        fields = ["id", "is_active", "nb_seats", "nb_available_seats", "start", "end"]
```

Comment:
Here the `is_active` field should be not the value of the field but the result of `is_active(order_group)` method